src/advanced/node-management.md

```diff
@@ -462,6 +334,157 @@ All explorer API endpoints share the same base path, denoted
 All explorer endpoints are public. `enable_blockchain_explorer` local
 configuration parameter allows to turn explorer endpoints on/off.
 
+### Transaction
+
+```none
+GET {explorer_base_path}/transactions/{transaction_hash}
+```
+
+Searches for a transaction, either committed or uncommitted, by the hash.
+
+#### Parameters
+
+- **transaction_hash**: Hash  
+  The hash of the transaction to be searched
+
+#### Response
+
+Returns a transaction from the pool of unconfirmed transactions if it is not
+committed yet, otherwise, returns a transaction from the blockchain.
+
+Response is a JSON object with one required field:
+
+- **type**: string  
+  Transaction type, can be:
+
+    - `committed`: committed transaction (in blockchain)
+    - `in-pool`: uncommitted transaction (in the pool of unconfirmed
+      transactions)
+    - `unknown`: unknown transaction
+
+##### Unknown Transaction
+
+Response JSON contains only `type` field. Its value is `unknown`. Additionally,
+returns HTTP status `404`.
+
+??? example "Response Example"
+    ```json
+    {
+      "type": "unknown"
+    }
+    ```
+
+##### Known Uncommitted Transaction
+
+Response JSON has same fields as [`SerializedTransaction`](#serializedtransaction)
+plus `type` field with value equal to `"in-pool"`.
+
+??? example "Response Example"
+    ```json
+    {
+      "type": "in-pool",
+      "body": {
+        "amount": "152",
+        "from": "b0d6af8bbe45c574c5f9dd8876b5b037b38d1bf861fd7b90744957aa608ed0c2",
+        "seed": "2953135335240383704",
+        "to": "99e396355cb2146aba0457a954ebdae36e09e3abe152693cfd1b9a0975850789"
+      },
+      "network_id": 0,
+      "protocol_version": 0,
+      "service_id": 128,
+      "message_id": 128,
+      "signature": "7d3c503d6dc02ca24faaeb37af227f060d0bcf5f40399fae7831eb68921fd00407f7845affbd234f352d9f1541d7e4c17b4cd47ec3f3208f166ec9392abd4d00"
+    }
+    ```
+
+##### Known Committed Transaction
+
+Response is a JSON object with the following fields:
+
+- **content**: SerializedTransaction  
+  Transaction with the specified hash
+- **location**: TransactionLocation  
+  Transaction position in the blockchain
+- **location_proof**: ListProof  
+  [Merkle proof](merkelized-list.md#merkle-tree-proofs) tying transaction
+  to the `tx_hash` of the containing block
+- **type**: `"committed"`  
+  Always equals to `committed`
+- **status**: Object  
+  [Transaction execution](../architecture/transactions.md#execute) status
+- **status.type**: `"success"` | `"error"` | `"panic"`  
+  Execution status kind:
+
+    - `"success"` denotes a successfully completed transaction
+    - `"error"` denotes a transaction that has returned an error (for example,
+      because of transaction parameters not satisfying context-dependent checks)
+    - `"panic"` denotes a transaction that has raised a runtime exception
```

Comment:
@DarkEld3r ,

Just wondering if it is OK to use "panic" (a Rust-specific term) in the context of polyglot Exonum? The other question is whether unrecoverable errors shall be treated as recoverable.

Reply:
@dmitry-timofeev
Good question. I think that we can use "panic", but not as a Rust-specific term. We can explain it as an Exonum transactions' specific term.

Reply:
OK, thanks, and what about always-attempting-to-recover Exonum? 

On top of that, I asked some more questions on the wiki page above, we can discuss this question there too.